tokio/src/runtime/task/stack.rs

```diff
@@ -1,83 +0,0 @@
-use crate::loom::sync::atomic::AtomicPtr;
-use crate::runtime::task::{Header, Task};
-
-use std::marker::PhantomData;
-use std::ptr::{self, NonNull};
-use std::sync::atomic::Ordering::{Acquire, Relaxed, Release};
-
-/// Concurrent stack of tasks, used to pass ownership of a task from one worker
-/// to another.
-pub(crate) struct TransferStack<T: 'static> {
```

Comment:
i guess the idea behind replacing this with another `LinkedList` is to reduce the number of unsafe data structures in tokio? but, this type is conceptually much simpler than the linked list. so, i'm a little unsure if replacing it with another linked list really makes tokio any safer or not.

:woman_shrugging:

Reply:
I mean, the number of uses of the linked list is unchanged by this diff, even if they are moved a bit around.

Reply:
ah, you're right --- turns out i can't read. carry on!